jest.config.js

```diff
@@ -17,7 +17,6 @@ module.exports = {
         '<rootDir>/node_modules/',
         '<rootDir>/test/specs/'
     ],
-    setupFiles: ['./jest.setup.js'],
```

Comment:
This duplicates functionality with line 21, the change is needed to be able to call `expect.extend` in jest.setup.js